packages/code-du-travail-modeles/src/modeles/conventions/1516_organismes_formation/__tests__/indemnite-licenciement/indemnite.spec.ts

```diff
@@ -7,23 +7,25 @@ const engine = new IndemniteLicenciementPublicodes(
 
 describe("Indemnité conventionnel de licenciement pour la CC 1516", () => {
   test.each`
-    seniority  | salary  | expectedCompensation
-    ${0}       | ${0}    | ${0}
-    ${2}       | ${1000} | ${0}
-    ${25 / 12} | ${1000} | ${416.67}
-    ${5}       | ${1000} | ${1000}
-    ${10}      | ${2000} | ${4000}
-    ${42}      | ${3000} | ${33300}
+    seniority  | seniorityRight | salary  | expectedCompensation
+    ${0}       | ${0}           | ${0}    | ${0}
+    ${18 / 12} | ${18 / 12}     | ${3037} | ${0}
+    ${2}       | ${2}           | ${3037} | ${0}
+    ${2.1}     | ${2}           | ${3037} | ${0}
+    ${2.1}     | ${2.1}         | ${3037} | ${1275.54}
+    ${23}      | ${23}          | ${3037} | ${16399.8}
+    ${26}      | ${26}          | ${3037} | ${18222}
+    ${42}      | ${42}          | ${3037} | ${18222}
```

Comment:
Top, par contre on peut challenger un peu :) 

```suggestion
    ${0}       | ${0}           | ${0}    | ${0}
    ${18 / 12} | ${18 / 12}     | ${3037} | ${0}
    ${2}       | ${3}           | ${3037} | ${0}
    ${2.1}     | ${2}           | ${3037} | ${0} <-- ce cas ne peut pas exister 🤔 
    ${2.1}     | ${2.1}         | ${3037} | ${1275.54}
    ${23}      | ${2.1}          | ${3037} | ${16399.8}
    ${26}      | ${2.1}          | ${3037} | ${18222}
    ${42}      | ${2.1}          | ${3037} | ${18222}
```

Reply:
la t'as échanger seniority et seniorityRigth ?
Mais effectivement c'était pas clair ce que je faisais je viens de le réécrire dis moi si tu aurai ajouté autre chose ?